k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-pod.yaml

```diff
@@ -66,9 +66,10 @@ spec:
         args:
           - -c
           {{- if .Values.cortxserver.machineid.value }}
-          - /opt/seagate/cortx/provisioner/bin/cortx_deploy -n $MACHINE_ID -f /etc/cortx/solution -c yaml:///etc/cortx/cluster.conf
+          - set -x;
```

Comment:
Necessary, or left over from debugging?

Reply:
Left over from the previous multi-line workaround for the stub entrypoint. Removed.